reactive/kubernetes_control_plane.py

```diff
@@ -2672,6 +2674,11 @@ def configure_scheduler():
         feature_gates.append("CSIMigrationGCE=false")
     elif is_state("endpoint.azure.ready"):
         feature_gates.append("CSIMigrationAzureDisk=false")
+    elif is_state("endpoint.vsphere.ready") and get_version("kube-apiserver") >= (
+        1,
+        12,
+    ):
```

Comment:
`(1, 12)` - Is this meant to be `(1, 21)`?

I'd just skip this check myself, the 1.25 charms won't support k8s < 1.23 anyway.

Reply:
Oh, I see, this is copied from the matching case in `configure_apiserver` and `configure_controller_manager` for consistency. Don't worry about it then, we can clean up these checks later.